Check for python2 support first, then python3. This should help avoid importing
the wrong module for python2 where the builtins module can be provided by the
futures package.

--- twine-1.11.0/tests/test_utils.py
+++ twine-1.11.0/tests/test_utils.py
@@ -19,9 +19,9 @@
 import textwrap
 
 try:
-    import builtins
-except ImportError:
     import __builtin__ as builtins
+except ImportError:
+    import builtins
 
 import pytest
 
